README.md

```diff
@@ -106,6 +106,17 @@ This generates a `CHANGELOG.md`, with pretty Markdown formatting.
 
 For further instructions please read the [GitHub Changelog Generator README](https://github.com/github-changelog-generator/github-changelog-generator#usage)
 
+### Rake Tasks
+
+Installe extension management rake tasks by adding this to the extension's `Rakefile`:
```

Comment:
This shouldn't simply be added to the `Rakefile`, it should replace what's in there, right?